packages/common/src/is-feature-active.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * @param {{ featureFlagging: { endpoint?: string, timeToLiveInMinutes: string | number } }} config
+ * @returns {(featureFlagName: string, localPlanningAuthorityCode?: string) => Promise<boolean>}
+ */
+exports.isFeatureActive = (config) => async (featureFlagName, localPlanningAuthorityCode) => {
```

Comment:
This code was in 3 packages, they now init their own instance from this by passing their configs

Reply:
good thinking 🧠

Reply:
I've done this now